libs/next_gen_ui_agent/types.py

```diff
@@ -219,6 +219,16 @@ class UIComponentMetadata(UIBlockComponentMetadata):
     Name of the field used for `JSON Wrapping` if it was performed, `None` if `JSON Wrapping` was not performed.
     """
 
+    # Debug information for LLM interactions
+    llm_interactions: Optional[list[dict[str, Any]]] = None
```

Comment:
We might probably make it part of the `UIBlockComponentMetadata` so they can be passed in the UI Agent response. Probably only if enabled in the UI Agent's configuration.  Might be usefull for the E2E app "Debug mode" feature/PR